### ISSUE TITLE
Strings allowed into Integer fields on Import

### ISSUE THREAD

**Issue by jacksonsox** [hankinsoft/SQLPro#587]
When using the import feature, string values are allowed to be imported into Integer value fields.  On export, the values remain as characters in the json output.  Processing strings in numeric fields programatically is costly and can result in errors if not trapped correctly.

**Comment by hankinsoft** [comment]
Hi,
Could you possible post your table structure along with an example of what you are importing? Just something small that will let me reproduce the issue if possible. Thanks!

**Comment by jacksonsox** [comment]
Thanks for looking at my query.

As a side note, it would be nice enhancement to allow importing of .txt files since tab files may not be .csv extensions.  Excel exports tab as .txt.  https://superuser.com/questions/577875/what-file-extension-does-a-tab-delimited-file-have/577912. Although, I understand that is a minor change for me to swap my file from .txt to .csv for each load.

File example.
[HasbroStarWarsBlack6In.txt](https://github.com/hankinsoft/SQLPro/files/5997621/HasbroStarWarsBlack6In.txt)

Table
<img width="331" alt="Screen Shot 2021-02-17 at 12 41 57 PM" src="https://user-images.githubusercontent.com/32544470/108245960-e6332080-711e-11eb-9ffd-aab37033c673.png">

**Comment by hankinsoft** [comment]
Just submitted 2021.16 which does a bit of data type detections on SQLite imports (also, I added txt as an option in the last build).

**Comment by jacksonsox** [comment]
This feature seems to be working better, but not completely correct.  When I did my export and then a full import, the following were allowed to import.  Although, all other fields seemed to work correctly.

The pattern that didn't appear to work were fields starting with numbers then followed by text.  I didn't do exhaustive testing.
<img width="194" alt="Screen Shot 2021-02-28 at 5 33 53 PM" src="https://user-images.githubusercontent.com/32544470/109436165-529a0380-79ec-11eb-9142-bc785149b99b.png">

**Comment by jacksonsox** [comment]
Actually, this issue still exists.  I just imported Now into an integer field.

**Comment by hankinsoft** [comment]
I've decided NOT to make any changes regarding this. SQLite allows importing non integer values into integer columns - after the initial changes I made with this when I attempted to do some validation I receive feedback from many users that the imports were no longer working as expected. As such, performing the validation would have breaking changes for existing users and also would NOT be the expected behaviour for SQLite.
